protocol/lavasession/provider_session_manager.go

```diff
@@ -159,7 +159,7 @@ func (psm *ProviderSessionManager) GetDataReliabilitySession(address string, epo
 
 func (psm *ProviderSessionManager) GetSession(ctx context.Context, address string, epoch uint64, sessionId uint64, relayNumber uint64) (*SingleProviderSession, error) {
 	if !psm.IsValidEpoch(epoch) { // fast checking to see if epoch is even relevant
-		utils.LavaFormatError("GetSession", InvalidEpochError, utils.Attribute{Key: "RequestedEpoch", Value: epoch}, utils.Attribute{Key: "blockedEpochHeight", Value: psm.blockedEpochHeight}, utils.Attribute{Key: "blockDistanceForEpochValidity", Value: psm.blockDistanceForEpochValidity})
+		utils.LavaFormatError("GetSessions", InvalidEpochError, utils.Attribute{Key: "RequestedEpoch", Value: epoch}, utils.Attribute{Key: "blockedEpochHeight", Value: psm.blockedEpochHeight}, utils.Attribute{Key: "blockDistanceForEpochValidity", Value: psm.blockDistanceForEpochValidity})
```

Comment:
@Aleksao998 , using replace all without validating what the change does is risky, always use F2 and rename the method. 
this is an example why its important to use F2 instead of replace all.

Reply:
Great catch, I haven't used the replace all, but rename variable from IDE, no idea why it changed all these. You can see in the code that there are many other _GetSession_ which were not changed, like the name of this method above. 

Anyways, it was my mistake not to validate all the changed entries. Thank you both for finding this 🙏🙏

Fixed e79a1b274b095bb14f4ac3262751b308cbf936b9